src/cmd/create.go

```diff
@@ -58,9 +58,10 @@ var (
 )
 
 var createCmd = &cobra.Command{
-	Use:   "create",
-	Short: "Create a new toolbox container",
-	RunE:  create,
+	Use:               "create",
+	Short:             "Create a new toolbox container",
+	RunE:              create,
+	ValidArgsFunction: completionEmpty,
```

Comment:
Isn't it enough to have `ValidArgs: nil` ?  Is it really necessary to have the dynamic variant?

Reply:
Yes, it is. If a custom completion function is not set, the default completion behaviour is then used -> listing of files/dirs in the current directory. `completionEmpty` ensures there is no completion shown. In the case of `create` that is the correct behaviour since it does not accept files/dirs as an argument but only an arbitrary string that the user has to choose.

Reply:
I see.  Thanks for explaining that!